--- TClasTool.h	2010-03-19 05:07:02.725013986 -0300
+++ TClasTool.h.mine	2010-03-19 05:08:16.477017883 -0300
@@ -174,7 +174,7 @@
       kHe3_Id,
       kHe4_Id};
 
-  static const char *fgParticle_Name[]=
+  static const char fgParticle_Name[][13] =
     {
       "Unknown",
       "Electron",
@@ -190,7 +190,7 @@
       "Helium3",
       "Helium4" };      
 
-  static const char *fgParticle_ShortName[]=
+  static const char fgParticle_ShortName[][13] =
     {
       "Unkn",
       "Elec",
@@ -206,7 +206,7 @@
       "He3",
       "He4" };      
 
-  static const char *fgParticle_Symbol[] =
+  static const char fgParticle_Symbol[][13] =
     {
       "?",
       "e^{-}",
